src/components/TableComponent/index.jsx

```diff
@@ -86,14 +86,15 @@ class TableComponent extends React.Component {
           <div className="col">
             {loading && <TableLoadingOverlay />}
             <div className="table-responsive">
-              <Table
+              <DataTable
```

Comment:
Similar to the suggestion in `StatusAlert`, what would it take to no longer have a custom `TableComponent/index.jsx` file/component, and instead only rely on `DataTable` directly where `TableComponent/index.jsx` was previously used?

This would be more tricky to do for tables than the alerts throughout the application, though, so I'd love to understand the level of effort involved and/or what functionality is in this `TableComponent/index.jsx` that can't be replicated with `DataTable` alone.

Reply:
yes it can be changed to DataTable but require much file changes, i would suggest we should do it in a separate PR to keep the scope of this PR intact